app/src/app/admin/abstractListingController.ts

```diff
@@ -53,11 +53,22 @@ namespace app.admin {
          * @returns {ng.IPromise<any[]>}
          */
         public search():ng.IPromise<any> {
-            return this.entitiesPaginator.complexQuery({
-                _all: [this.queryString],
-                authorId: _.pluck(this.usersToFilter, 'userId'),
-                _tags: {tagId:_.pluck(this.tagsToFilter, 'tagId')}
-            })
+
+            let query = {};
```

Comment:
It would be better to define an interface for the query so that you can use direct dot assignment rather than the somewhat flaky string accessor assignment.